exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/flatten/FlattenRecordBatch.java

```diff
@@ -99,7 +100,23 @@ private void clear() {
     }
   }
 
-  private class FlattenMemoryManager extends AbstractRecordBatchMemoryManager {
+  public enum Metric implements MetricDef {
+    INPUT_BATCH_COUNT,
+    AVG_INPUT_BATCH_BYTES,
+    AVG_INPUT_ROW_BYTES,
+    TOTAL_INPUT_RECORDS,
+    OUTPUT_BATCH_COUNT,
+    AVG_OUTPUT_BATCH_BYTES,
+    AVG_OUTPUT_ROW_BYTES,
+    TOTAL_OUTPUT_RECORDS;
```

Comment:
Very nice, these labels should be quite readable in the UI. (We should have an enum-to-label lookup table, but we don't...)

This pass I noticed the inconsistency between `INPUT_BATCH_COUNT` and `TOTAL_INPUT_RECORDS`. Perhaps change the latter to `INPUT_RECORD_COUNT`.